### PR TITLE
Add print logs and cleaning up cache in regression-repo…

### LINKED ISSUE #15606: Add print logs and cleaning up cache in regression-report.yml
Add new step to regression-report.yml in the beginning to clean cache and also print out the cache files being stored in generate_report.

### PR DESCRIPTION
**Pull request by relentless-pursuit**
Resolves #15606:

### CODE REVIEW

**Comment by romani** [comment]
Please fix lint for GitHub action

**Comment by romani** [comment]
I don't like this magic.
Let's not do cleaning of such all caches.

If we generate files properly, we don't care what is cache at all. All we care is content and specific files.

**Comment by relentless-pursuit** [comment]
> I don't like this magic. Let's not do cleaning of such all caches.
> 
> If we generate files properly, we don't care what is cache at all. All we care is content and specific files.

i changed cache keys. this should work.

File: .github/workflows/regression-report.yml [hunk 1]
```diff
@@ -496,6 +496,17 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: ${{ env.AWS_REGION }}
 
+      - name: Print config.xml
+        run: |
+          echo "Contents of config.xml:"
+          echo "-------------------------"
+          if [ -f .ci-temp/contribution/checkstyle-tester/diff_config.xml ]; then
+            cat .ci-temp/contribution/checkstyle-tester/diff_config.xml
```
Comment (romani) [inline .github/workflows/regression-report.yml:504]: Please do this also for list of projects

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: .github/workflows/regression-report.yml (hunk 1 above)
Comment (relentless-pursuit) [reply .github/workflows/regression-report.yml:504]: done

**Review by romani — APPROVED** [review]

**Comment by romani** [comment]
@relentless-pursuit , in next PR please send update to clean/delete content of cache at the end workflow, to not generate mess and keep cache to be light to mount to execution.

**Pull request merged**